ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__automated-release-stable-4.21-upgrade-from-stable-4.20.yaml

```diff
@@ -0,0 +1,105 @@
+base_images:
+  ansible:
+    name: "4.20"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.20"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.21"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.20"
+  tools:
+    name: "4.20"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+  vsphere-ci-python:
+    name: vsphere-python
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.20"
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.21.0-0
+        stream: 4-dev-preview
+        upper: 4.22.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-f999
+  cron: 8 8 29 2 *
```

Comment:
@rioliu-rh I see this  `cron: 8 8 29 2 *`  in multiple places in the config files for our autorelease jobs. If my understanding about this pattern is right then this indicates that then "This cron job will only run on leap years, because February 29 occurs only in leap years." Is that intentional?

Reply:
the cron setting means, this job won't be triggered by scheduler. we only trigger auto-release job via Gangway API